ESPMaster/ESPMaster.ino

```diff
@@ -236,8 +244,26 @@ void setup() {
 
   if (isWifiConfigured && !isPendingReboot) {
     //ezTime initialization
+
+    if (strlen(timezoneServer) > 0) {  // Check if timezoneServer is not empty
+        setServer(timezoneServer);
+    } else {
+        SerialPrintln("No timezone server specified, skipping setServer()");
+    }
```

Comment:
Wanted to make the small suggesting of updating the indentation to just keep it in line with the rest of the file if possible

```suggestion
    if (strlen(timezoneServer) > 0) {  // Check if timezoneServer is not empty
      setServer(timezoneServer);
    } else {
      SerialPrintln("No timezone server specified, skipping setServer()");
    }
```